paddle/operators/recurrent_op.h

```diff
@@ -117,10 +130,14 @@ class RecurrentOp final : public framework::OperatorBase {
     alg_.Run(scope, dev_ctx);
   }
 
+  void set_stepnet(std::shared_ptr<NetOp> net) { stepnet_ = net; }
+  const NetOp* stepnet() const { return stepnet_.get(); }
+
   static const rnn::ArgumentName kArgName;
 
  private:
   RecurrentAlgorithm alg_;
+  std::shared_ptr<NetOp> stepnet_;
```

Comment:
Do you mean something like [here](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/framework/pybind.cc#L230):

```cpp
py::class_<operators::NetOp, std::shared_ptr<operators::NetOp>> net(m, "Net");
```

It seem that calling Python `new Net()` returns a `shared_ptr<operator::NetOp>` -- however, when will the reference count become 0 and the C++ instance get freed?